scalameta/io/js/src/main/scala/scala/meta/internal/io/PlatformPathIO.scala

```diff
@@ -2,28 +2,28 @@ package scala.meta.internal.io
 
 import scala.meta.io._
 
-object PathIO extends CommonPathIO {
+object PlatformPathIO {
+  private [io] def isNode = JSFs != null
```

Comment:
This object is internal and in a Platform* file, what object do you think would be better?